linkerd/app/src/env.rs

```diff
@@ -448,7 +455,20 @@ pub fn parse_config<S: Strings>(strings: &S) -> Result<super::Config, EnvError>
             } else {
                 outbound.proxy.connect.clone()
             };
+
+            // Add any extra Labels, that are not present in the labels file
+            let mut labels: HashMap<String, String> = [(
+                "namespace".to_string(),
+                namespace.unwrap_or_default().unwrap_or_default(),
+            )]
+            .iter()
+            .cloned()
+            .collect();
```

Comment:
What's this doing, just creating a Map? Isn't this more code than just instantiating it and inserting the value?

```rust
let mut labels = HashMap::new();
labels.insert("namespace".to_string(), namespace.flatten().unwrap_or_default())
```

Keep it simple.